core/src/test/java/kafka/test/ClusterConfigTest.java

```diff
@@ -96,4 +98,16 @@ public void testDisksPerBrokerIsZero() {
                 .setDisksPerBroker(0)
                 .build());
     }
+
+    @Test
+    public void testDisplayTags() {
+        List<String> tags = Collections.singletonList("tag");
```

Comment:
Could you please use multi tags and then verify all of them are existent in `displayTags`?

Reply:
@chia7712 I addressed multi tags in my latest commit.